WiserTaskScheduler/WiserTaskScheduler/Modules/Slack/modules/SlackSettings.cs

```diff
@@ -1,4 +1,5 @@
 ﻿namespace WiserTaskScheduler.Modules.Slack.modules;
+using Microsoft.Extensions.Logging;
```

Comment:
The using is the only change (left) in this file. Is it still needed?

Reply:
No, its not, nice spot, thank you